dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry10/internal/rest/TelemetryServletFilter.java

```diff
@@ -101,14 +99,17 @@ private Instrumenter<ServletRequest, ServletResponse> getInstrumenter() {
     }
 
     private Instrumenter<ServletRequest, ServletResponse> createInstrumenter() {
+        // Check if the HTTP tracing should be disabled
+        boolean httpTracingDisabled = config.getOptionalValue("otel.trace.http.disabled", Boolean.class).orElse(false);
```

Comment:
Should use the `CONFIG_DISABLE_HTTP_TRACING_PROPERTY` constant that the removed code used for the config key.